lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -322,6 +322,36 @@ serve as complete descriptions of the runtime of a tool.
       <xs:element name="requirement" type="Requirement" minOccurs="0" maxOccurs="unbounded"/>
       <xs:element name="container" type="Container" minOccurs="0" maxOccurs="unbounded"/>
     </xs:sequence>
+    <xs:attribute name="cores_min" type="xs:long" use="required">
```

Comment:
`use="required"` is making it mandatory to specify a value if the attribute is set, or?